json/schema/user_import.schema.json

```diff
@@ -0,0 +1,191 @@
+{
```

Comment:
Don't we require `company_id` as well? Without it we wouldn't know where to put user to : >

Reply:
Also I think we require `company_name` regardless of what type of import it is, at least based on https://github.com/workivate/onboarding-file-formats#properties

Reply:
This is meant for user-only uploads where all employees in the file belong to a single company and the company details are retrieved from another source, for example in the admin panel the company details will depend on which company is the user managing when they use the upload feature to manage employee accounts. I'll update the README to explain this.